source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -384,9 +423,9 @@ Mapping options
 
 There are various option that can be used to change some behavior:
 
-============================================================================================== =========================================== ================================================================================================================= ===================================================================================== ================================================================================================================================================================================================================
+============================================================================================== =========================================== ================================================================================================================= ===================================================================================== ======================================================================================================================================================================================================================================
 Setting                                                                                        Variable value                              Default                                                                                                           Range                                                                                 Description
```

Comment:
My experience with this table:
* the content is very wide and I have to scroll horizontally to check the description for example
* I don't need the `Variable value` column. If I'd be interested, I'd look at the settings in the js file
-> this would reduce the table width
* in the `Default` column we may remove the the prefix (LEDColors etc.) and just show the values
-> this would reduce the table width